src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
```

Comment:
Please use singular nouns for naming tables

Reply:
> Please use singular nouns for naming tables

Based on the general structure of the project, the names for other tables are used in the plural, I think it would be incorrect to use the singular for this case